code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -106,15 +106,16 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/storage/secure/briefcase = 3,
 	/obj/item/storage/toolbox/artistic = 2,
 	/obj/item/toy/eightball = 1,
-	/obj/item/reagent_containers/pill/floorpill = 1,
+	/obj/item/reagent_containers/pill/floorpill = 4,
 	/obj/item/reagent_containers/food/snacks/cannedpeaches/maint = 2,
-	/obj/item/storage/daki = 3, //VERY IMPORTANT CIT CHANGE - adds bodypillows to maint
-	/obj/item/storage/pill_bottle/penis_enlargement = 2,
-	/obj/item/storage/pill_bottle/breast_enlargement = 2,
+	/obj/item/storage/daki = 1, //VERY IMPORTANT CIT CHANGE - adds bodypillows to maint
+	/obj/item/clothing/glasses/hud/diagnostic = 1,
+    /obj/item/clothing/glasses/hud/health = 1,
 	/obj/item/clothing/shoes/wheelys = 1,
 	/obj/item/clothing/shoes/kindleKicks = 1,
-	/obj/item/autosurgeon/penis = 1,
-	/obj/item/autosurgeon/testicles = 1,
+	/obj/item/storage/backpack/duffelbag/syndie = 1
+	/obj/item/storage/belt/fannypack =1,
```

Comment:
missing space before the 1

Reply:
wait
>/obj/item/storage/backpack/duffelbag/syndie
this might cause some confusion but the good kind.

Reply:
Turned out fannypacks couldn't work on maintenance loot even when fixed.

/shrug